src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -169,52 +164,67 @@ internal static TimeSpan ResolveTimeSpanFromMilliseconds(string nameOfProperty,
 
         private string ResolveAttributeConnectionString(CosmosDBTriggerAttribute attribute)
         {
-            string connectionString = this._monitorConnectionString;
-            if (!TryReadFromSettings(attribute.ConnectionStringSetting, connectionString, out connectionString))
```

Comment:
I think `TryReadFromSettings` is no longer used but still in the code

Reply:
Oops yep you're right. I'll pull it out.